.eslintrc.js

```diff
@@ -30,22 +30,53 @@ module.exports = {
         objects: 'always-multiline',
         imports: 'always-multiline',
         exports: 'always-multiline',
-        functions: 'ignore',
+        functions: 'never',
       },
     ],
     'no-prototype-builtins': 0,
     'prefer-destructuring': 0,
-    'no-else-return': 0,
+    'no-else-return': 1,
     'lines-between-class-members': ['error', 'always', { exceptAfterSingleLine: true }],
     '@typescript-eslint/explicit-member-accessibility': 0,
     '@typescript-eslint/no-explicit-any': 0,
     '@typescript-eslint/no-inferrable-types': 0,
     '@typescript-eslint/explicit-function-return-type': 0,
     '@typescript-eslint/no-use-before-define': 0,
     '@typescript-eslint/no-empty-function': 0,
-    'curly': ['error', 'all']
+    'curly': ['error', 'all'],
+    'padding-line-between-statements': [
```

Comment:
Wow-wow it's amazing 👍
Definitely will add the same rules in our internal projects! 😍

Reply:
i searched for an rule similar to `newline-before-return` from tslint (to have consistent returns here) and the first one i found was deprecated into this rule which was more capable - so i also added more stuff an just the return thing